meshmode/mesh/__init__.py

```diff
@@ -497,6 +498,35 @@ class TensorProductElementGroup(_ModepyElementGroup):
 # }}}
 
 
+# {{{ volume groups
+
+@dataclass(frozen=True)
+class VolumeGroup:
```

Comment:
```suggestion
class TaggedElementSubgroup:
```
?
Or `ElementSubgroup`?